src/components/HomeNav.js

```diff
@@ -29,8 +40,18 @@ class HomeNav extends Component {
               </div>
               <a className='login-nav' href='#'>LOGIN</a>
               <a className='demo-button' href='#'>BOOK A DEMO</a>
-              <input className='trial-input' type='text' placeholder='Email address' />
-              <a className='nav-trial-button' href='#'>Free Trial</a>
+              <form onSubmit={this.handleSubmit} action='/register'>
+                <input
+                  className='trial-input'
+                  name="email"
+                  type='text'
+                  placeholder='Email address'
+                  onChange={this.handleChange.bind( this )} />
```

Comment:
Instead of using .bind( this ), make handeChange an arrow function.` handleChange = ( event ) => {}` Arrow functions are already bound by nature.